### PR TITLE
Add dropdown styling

### PR DESCRIPTION
**Pull request by caproven**
Adds support for styling of dropdowns. Adapts the changes from upstream tview (https://github.com/rivo/tview/issues/621) to this fork.

Enables us to address https://github.com/derailed/k9s/issues/1625.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -7,7 +7,6 @@ require (
 	github.com/lucasb-eyer/go-colorful v1.2.0
 	github.com/mattn/go-runewidth v0.0.12
 	github.com/rivo/uniseg v0.2.0
-	github.com/stretchr/testify v1.7.0
```
Comment (slimus) [inline go.mod:10]: Why did you change `go.mod` and `go.sum`? Can you revert back it, please?

Comment (caproven) [reply go.mod:10]: I had a linter warning that testify was unused. The `go.mod` & `go.sum` changes are the result of a `go mod tidy`. I have no qualms about reverting this change but don't see a reason to keep an unused dependency.
Shall we split it into a separate commit/PR, or leave it as is?

Comment (slimus) [reply go.mod:10]: No, you're good. 
@derailed LGTM

**Comment by derailed** [comment]
Cool! Thank you both for the work and review!
